### DISCUSSION TITLE
Shared code library for Uno UWP and WinUI project.

### DISCUSSION THREAD

**Discussion by DmitryBorodiy** [unoplatform/uno#15750] [Ideas]
Hello, Uno Platform community!
I need UWP support for my project, but I also need support for newer versions of .NET for non-Windows platforms for best productivity. What can I do?

Thanks in advance!

**Comment by jeromelaban** [comment]
Thanks for the question! This particular scenario is getting more difficult as years go by (particularly because of C# versions compatibility), but you can use the Uno Platform UWP template to get you going.
